awsutil/generate_credentials.go

```diff
@@ -180,6 +180,14 @@ func (c *CredentialsConfig) GenerateCredentialChain(opt ...Option) (*credentials
 		providers = append(providers, webIdentityProvider)
 	}
 
+	profile := os.Getenv("AWS_PROFILE")
```

Comment:
Since this is only used in the shared creds provider, can you move this into the `opts.withSharedCredentials` block below?

Reply:
@jefferai Fixed!